packages/reactive-element/src/decorators/base.ts

```diff
@@ -44,3 +46,66 @@ export const standardPrototypeMethod = (
   key: element.key,
   descriptor,
 });
+
+/**
+ * Helper for decorating a property that is compatible with both TypeScript
+ * and Babel decorators. The optional `finisher` can be used to perform work on
+ * the class. The optional `descriptor` should return a PropertyDescriptor
+ * to install for the given property.
+ *
+ * @param finisher {(ctor: typeof ReactiveElement, property: PropertyKey) => void)} Optional finisher
+ * @param descriptor {(property: PropertyKey) => PropertyDescriptor} Optional descriptor generator
```

Comment:
Not sure if that's the right way to write jsdoc for destructured params like that...

Reply:
Edited.